src/tools/html-checker/main.rs

```diff
@@ -0,0 +1,96 @@
+use std::env;
+use std::path::Path;
+use std::process::{Command, Output};
+
+fn check_html_file(file: &Path) -> usize {
+    let to_mute = &[
+        // "disabled" on <link> or "autocomplete" on <select> emit this warning
+        "PROPRIETARY_ATTRIBUTE",
+        // It complains when multiple in the same page link to the same anchor for some reason...
```

Comment:
Yes, because this is a bug and randomizes which anchor it will pick: https://github.com/rust-lang/rust/issues/76895

Reply:
No no, not the same id, the same href.